pages/account/how-to/use-2fa.mdx

```diff
@@ -3,98 +3,117 @@ title: How to use Multifactor Authentication (MFA)
+## How to enable passkeys
+
+A passkey is a passwordless authentication method that allows you to securely log in to apps and websites through public-key cryptography. Passkeys at Scaleway are supported by the WebAuthn standard.
 
-MFA is now disabled on your account.
+When you enable a passkey, your password manager or device generates a public/private key pair. The private key stays securely on your device while the public key is sent to Scaleway.
 
-## How to regain access to your account
+When you log in, the Scaleway sends a challenge to your device. Your device uses the private key to sign the challenge. Scaleway verifies the response using the public key.
 
-### If you lose your MFA device
+<Message type="important">
+  This MFA method is currently not available for IAM Members.
+</Message>
 
-If you lose your authentication device, you can regain access to your account using the backup codes that were generated when you enabled MFA.
+<Requirements />
 
-When you log into your account, you are prompted for an MFA code.
+- [Owner](/iam/concepts/#owner) status
+- Set up a [password manager](/iam/concepts/#password-manager) on your device
 
-Enter one of your **backup codes** instead of a code generated by your MFA device in real-time, then click **Log in**. You are logged into your account and can [disable MFA](#how-to-disable-mfa) if you wish.
+1. Go to your [Account Security](https://console.scaleway.com/settings/account/security) page.
+2. Click **+ Add passkey**, in the **Multifactor authentication** section. A pop-up displays.
+3. Enter the passkey name you configured in your password manager.
+4. Click **Start**. Your password manager will be called and a pop-up might appear in your browser.
+5. Follow your password manager's instructions to confirm and finish setting up the passkey.
+    If the process was successful, you will see a pop-up affirming that the **passkey was added**.
+6. Click **Close**.
 
-### If you lose your MFA device and backup codes
+You see a list of your passkeys.
 
-If you lose both your MFA app and backup codes, the only way to regain access to your account is through a manual identity verification.
+### How to delete a passkey
 
-To do so, make a request to our Trust and Safety team. You must make your request from the e-mail address registered on your account. In your email, provide the following information:
+1. Go to your [Account Security](https://console.scaleway.com/settings/account/security) page.
+2. Click <Icon name="delete" />, next to the passkey you want to delete in the **Multifactor authentication** section. A pop-up displays.
+3. Type **DELETE** in the box to confirm, then click **Delete passkey**.
 
-- The telephone number registered on your account
-- If you have an individual account, your passport or local ID card
-- If you have a corporate account, the certificate of incorporation and a copy of the last Scaleway invoice you received
+TOTP is now disabled on your account.
```

Comment:
I think this is webauthn, but it is not alway disabled (if i had 3 passkeys and delete one, mfa is not disabled)

It will only disable mfa if you delete your last passkey and don't have a totp